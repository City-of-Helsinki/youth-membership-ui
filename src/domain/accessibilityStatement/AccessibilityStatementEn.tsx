import React, { Fragment } from 'react';

import Text from '../../common/components/text/Text';

function AccessibilityStatementEn() {
  return (
    <Fragment>
      <Text variant="h1">Accessibility Statement</Text>
      <p>
        This accessibility statement applies to the website Youth membership
        registration. The site address is https://jassari.hel.fi.
      </p>

      <Text variant="h2">Statutory provisions applicable to the website</Text>
      <p>
        This website was published after 23 September 2018. The website must
        fulfil statutory accessibility requirements.
      </p>

      <Text variant="h2">The objective of the city</Text>
      <p>
        As regards the accessibility of digital services, Helsinki aims to reach
        at least Level AA or above as set forth in the WCAG guidelines in so far
        as is reasonably practical.
      </p>

      <Text variant="h2">Compliance status</Text>
      <p>
        The accessibility compliance of this website has not yet been evaluated.{' '}
      </p>

      <Text variant="h2">Preparing an accessibility statement</Text>
      <p>This statement was prepared on 6.4.2020</p>

      <Text variant="h2">Updating the accessibility statement</Text>
      <p>
        The accessibility statement will be updated so that it corresponds with
        the observations related to accessibility compliance made during an
        audit.
      </p>

      <Text variant="h2">Requesting information in an accessible format</Text>
      <p>
        If a user feels that content on a website is not available in an
        accessible format, they can request for this information by e-mail at{' '}
        <a href="mailto:helsinki.palaute@hel.fi">helsinki.palaute@hel.fi</a> or
        through the feedback form at{' '}
        <a href="https://www.hel.fi/feedback">www.hel.fi/feedback</a>. The aim
        is to reply to the enquiry within a reasonable time frame.
      </p>

      <Text variant="h2">Feedback and contact information</Text>
      <p>
        City of Helsinki, Culture and Leisure Division
        <br />
        Youth Services
        <br />
        <a href="mailto:nuorisopalvelut@hel.fi">nuorisopalvelut@hel.fi</a>
      </p>

      <Text variant="h2">The City of Helsinki and accessibility</Text>
      <p>
        The objective of the city of Helsinki is to be an accessible city to
        all. Helsinki aims to ensure that all residents are able to move about
        and act as effortlessly as possible and that all content and services
        are accessible to all.
      </p>
      <p>
        The city promotes accessibility of digital services by streamlining
        publishing work and organising accessibility-related training for its
        staff.
      </p>
      <p>
        The accessibility level of websites is monitored constantly during their
        maintenance. Immediate action will be taken if deficiencies are found.
        The aim is to carry out the necessary amendments as quickly as possible.
      </p>

      <Text variant="h2">The disabled and users of assistive technologies</Text>
      <p>
        The city provides counselling and support for the disabled and users of
        assistive technologies. Support is available on guidance sites announced
        on the cityâ€™s website and through telephone counselling.{' '}
      </p>

      <Text variant="h2">Approval of the accessibility statement</Text>
      <p>This statement was approved on 6.4.2020</p>
      <p>
        Culture and Leisure Division
        <br />
        City of Helsinki
      </p>
    </Fragment>
  );
}

export default AccessibilityStatementEn;
