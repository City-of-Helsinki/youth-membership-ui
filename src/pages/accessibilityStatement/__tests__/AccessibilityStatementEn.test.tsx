import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import AccessibilityStatementEn from '../AccessibilityStatementEn';

test('matches snapshot', () => {
  const wrapper = shallow(<AccessibilityStatementEn />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
