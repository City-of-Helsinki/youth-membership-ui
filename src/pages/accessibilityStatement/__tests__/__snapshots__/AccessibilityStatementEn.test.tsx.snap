// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Fragment>
  <h1>
    Accessibility Statement
  </h1>
  <p>
    This accessibility statement applies to the website Youth membership registration. The site address is https://jassari.hel.fi.
  </p>
  <h2>
    Statutory provisions applicable to the website
  </h2>
  <p>
    This website was published after 23 September 2018. The website must fulfil statutory accessibility requirements.
  </p>
  <h2>
    The objective of the city
  </h2>
  <p>
    As regards the accessibility of digital services, Helsinki aims to reach at least Level AA or above as set forth in the WCAG guidelines in so far as is reasonably practical.
  </p>
  <h2>
    Compliance status
  </h2>
  <p>
    The accessibility compliance of this website has not yet been evaluated.
     
  </p>
  <h2>
    Preparing an accessibility statement
  </h2>
  <p>
    This statement was prepared on 6.4.2020
  </p>
  <h2>
    Updating the accessibility statement
  </h2>
  <p>
    The accessibility statement will be updated so that it corresponds with the observations related to accessibility compliance made during an audit.
  </p>
  <h2>
    Requesting information in an accessible format
  </h2>
  <p>
    If a user feels that content on a website is not available in an accessible format, they can request for this information by e-mail at
     
    <a
      href="mailto:helsinki.palaute@hel.fi"
    >
      helsinki.palaute@hel.fi
    </a>
     or through the feedback form at
     
    <a
      href="https://www.hel.fi/feedback"
    >
      www.hel.fi/feedback
    </a>
    . The aim is to reply to the enquiry within a reasonable time frame.
  </p>
  <h2>
    Feedback and contact information
  </h2>
  <p>
    City of Helsinki, Culture and Leisure Division
    <br />
    Youth Services
    <br />
    <a
      href="mailto:nuorisopalvelut@hel.fi"
    >
      nuorisopalvelut@hel.fi
    </a>
  </p>
  <h2>
    The City of Helsinki and accessibility
  </h2>
  <p>
    The objective of the city of Helsinki is to be an accessible city to all. Helsinki aims to ensure that all residents are able to move about and act as effortlessly as possible and that all content and services are accessible to all.
  </p>
  <p>
    The city promotes accessibility of digital services by streamlining publishing work and organising accessibility-related training for its staff.
  </p>
  <p>
    The accessibility level of websites is monitored constantly during their maintenance. Immediate action will be taken if deficiencies are found. The aim is to carry out the necessary amendments as quickly as possible.
  </p>
  <h2>
    The disabled and users of assistive technologies
  </h2>
  <p>
    The city provides counselling and support for the disabled and users of assistive technologies. Support is available on guidance sites announced on the cityâ€™s website and through telephone counselling.
     
  </p>
  <h2>
    Approval of the accessibility statement
  </h2>
  <p>
    This statement was approved on 6.4.2020
  </p>
  <p>
    Culture and Leisure Division
    <br />
    City of Helsinki
  </p>
</Fragment>
`;
